joblib/test/test_memory.py

```diff
@@ -569,8 +558,16 @@ def test_memory_in_memory_function_code_change(tmpdir):
 
 
 def test_clear_memory_with_none_cachedir():
-    mem = Memory(cachedir=None)
-    mem.clear()
+    memory = Memory(cachedir=None)
+    memory.clear()
+
+
+def func_with_kwonly_args(a, b, kw1='kw1', kw2='kw2'):
```

Comment:
For completeness, the flake8 is done on the diff that the PR introduces. The main reason is historical: there are some flake violations in some files and we don't want to fix them all because it will introduce conflicts in ongoing PRs. On the other hand we don't want that PRs introduce additional flake8 violations.

Because the flake8 is done on the diff, if you don't change a line that already has a flake8 violation, Travis will be green. In this case we were changing a line that had a flake8 violation so it is in the diff and Travis is red.